src/ethereum_test_tools/rpc/eth_rpc.py

```diff
@@ -0,0 +1,66 @@
+"""
+Ethereum `eth_X` JSON-RPC methods used within EEST based hive simulators.
+"""
+
+from typing import Dict, List, Literal, Union
+
+from ..common import Address
+from .base_rpc import BaseRPC
+
+BlockNumberType = Union[int, str, Literal["latest", "earliest", "pending"]]
```

Comment:
I don't think we need `str` here - it's covered by the string literals?
https://docs.infura.io/networks/ethereum/json-rpc-methods/eth_getblockbynumber
```suggestion
BlockNumberType = Union[int, Literal["latest", "earliest", "pending"]]
```

Reply:
I had to add this for tox, otherwise it complains

Reply:
Found a better solution in: https://github.com/danceratopz/execution-spec-tests/pull/27/commits/f72268e0084d833c84f0c65e2b0f0d8941282678